### PR TITLE
Feat: P0 features

### PR DESCRIPTION
**Pull request by Rohitprasad83**
- A good user onboarding
- Display time, current weather in sync
- Wallpaper and quote changing on every refresh
- Published on chrome webstore
- Add your main focus for today.

--- 

image
image
image
image

### CODE REVIEW

**Review by hiteshverma27 — APPROVED** [review]
code looks good
- folder structure is good
- good naming conventions used 
- UI looks good

**Review by ranitasahaa1230 — APPROVED** [review]
Good use of contexts and reducers
The folder structure is proper
Great job with the tailwind
Functionality is working fine

File: src/reducer/userReducer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                city: action.payload,
+            }
+        case 'STORE_NAME':
+            localStorage.setItem('name', state.name)
+            return state
+        case 'STORE_CITY':
+            localStorage.setItem('city', state.city)
+            return state
+        default:
+            return {...state }
```
Comment (ranitasahaa1230) [inline src/reducer/userReducer.js:20]: Instead of returning state in the default, you can also throw an error

**Review by ranitasahaa1230 — APPROVED** [review]

File: src/reducer/userReducer.js (hunk 1 above)
Comment (Rohitprasad83) [reply src/reducer/userReducer.js:20]: will do it

**Pull request merged**